components/search_engines/pref_names.h

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
what about extending `search_engines_pref_names.h`? less conflict.

Reply:
done.

Reply:
I thought about extending chromium file via `chromium_src/components/search_engines/search_engines_pref_names.h`.
That way you don't need to add `brave_components_search_engines_sources` in `.gn`.

Reply:
@goodov  Oops! :) Addressed by https://github.com/brave/brave-core/pull/8626/commits/131c9aabd5e5bef4c60264b53107baac839dcab8